apps/ewallet_db/lib/ewallet_db/transaction_request.ex

```diff
@@ -141,6 +145,9 @@ defmodule EWalletDB.TransactionRequest do
         :require_confirmation,
         :max_consumptions,
         :max_consumptions_per_user,
+        :max_consumptions_per_interval,
```

Comment:
Can we validate that `:max_consumptions_per_interval` and `:max_consumptions_per_interval_per_user` can be set if `:consumption_interval_duration` is set? 

It'll be a headache to support users who only fill in partially.

Reply:
ได้ 👍